test/regressions/site/src/tests/Chip/DeletableAvatarChip.js

```diff
@@ -0,0 +1,16 @@
+// @flow weak
+
+import React from 'react';
+import Avatar from 'material-ui/Avatar';
+import Chip from 'material-ui/Chip';
+
+export default function DeletableAvatarChip() {
+  return (
+    <Chip
+      avatar={<Avatar>MB</Avatar>}
+      label="SvgIcon Chip"
+      onClick={() => {}}
```

Comment:
Right, no - copy pasted from the docs demo.  Although that makes me curious - is it possible to regression test the hover / focus / active styles?